src/Entities/Car.php

```diff
@@ -0,0 +1,34 @@
+<?php
+class Car 
+{
+    public int $id;
+    public string $model;
+    public int $make_id;
+    public string $make;
+    public int $bodytype_id;
+    public string $bodytype;
+    public ?string $image = null;
+    public int $year;
+    
+    public function __construct(
+        int $id, 
+        string $model, 
+        int $make_id,
+        string $make,
+        int $bodytype_id,
+        string $bodytype,
+        ?string $image,
+        int $year
+    )
+    {
+        $this->id = $id;
+        $this->model = $model;
+        $this->make_id = $make_id;
+        $this->make = $make;
+        $this->bodytype_id = $bodytype_id;
+        $this->bodytype = $bodytype;
+        $this->image = $image;
```

Comment:
Since you've made image nullable, the best thing to do is make it the last param, with a default of null. That way you don't need to explicitely pass in `null`, you can just not pass in anything at all.